roles/firewall/tasks/main.yml

```diff
@@ -71,6 +71,13 @@
   tags:
     - firewall
     - monitoring
+    
+- name: Allow ZMQ connections
+  ufw:  rule=allow
+        port=5550
+        proto=tcp
```

Comment:
Yes it's a concern and I don't know the right answer as yet. Currently ZMQ pushes from LAVA don't support encryption, I've just asked for it.

Either we run a single zmq reactor on a kci server and let every lab push to it, or every lab runs their own zmq reactor, or some mixture of both. I'm on a dynamic IP so it can't be locked down for me.

If every lab runs their own reactor, there's an admin overhead of getting it setup, and also keeping the reactor job parser script updated on all labs.

Maybe static ip labs can push to the kci reactor (so that the port can be ip secured) and dynamic ip labs like mine will have to run their own. I should probably put this in the meeting discussion document.